nautobot_golden_config/nornir_plays/config_deployment.py

```diff
@@ -67,7 +68,8 @@ def config_deployment(job_result, data, commit):
     now = datetime.now()
     logger = NornirLogger(__name__, job_result, data.get("debug"))
     logger.log_debug("Starting config deployment")
-
+    logger.log_debug(f"{dir(job_result)}")
+    logger.log_debug(f"{type(job_result)}")
```

Comment:
```suggestion
```

Are these still needed? These seem like testing messages.

Reply:
thanks miss to removing them.